Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ruby:2.7.2
+LABEL author="GetODK Inc."
+LABEL maintainer="ODK Build maintainers"
+LABEL description="ODK Build, a drag-and-drop Xform designer"
+
+# Install system dependencies
+RUN curl -sL https://deb.nodesource.com/setup_14.x | bash - 
+RUN DEBIAN_FRONTEND=noninteractive apt-get update \
+  && DEBIAN_FRONTEND=noninteractive apt-get install --yes \
+  -o Acquire::Retries=10 --no-install-recommends \
+  nodejs libpq-dev zlib1g-dev libbz2-dev default-jre \
```

Comment:
what's the jre for?

Reply:
It's a Ruby gem dependency. Eww, I know. I think it's the minifier/build thing. Tom and I stumbled over it a few months back.